metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -367,6 +376,17 @@ public TableDto deleteAndReturn(final QualifiedName name, final boolean isMView)
             }
         }
 
+        if (!config.isParentChildDropEnabled()) {
+            if (parentChildRelMetadataService.isChildTable(name)) {
+                throw new RuntimeException(name + " is a child table and "
+                    + "dropping a child table is currently disabled");
+            }
+            if (parentChildRelMetadataService.isParentTable(name)) {
```

Comment:
Regardless of this check the dropping of parent table should be disabled anyways right